### PR TITLE
Add spieren voor spieren

### PR DESCRIPTION
**Pull request by rubenpeters91**
### Added
- Added a small agenda "Spieren voor Spieren" outside of the RCT

### Changed
- Changed treatment group to 2 for agendas outside RCT
- Removed unnecessary subagendas for Longziekten and Poli-blauw

See test-dashboard: https://rsc.ds.umcutrecht.nl/content/a4f403a9-8d43-4757-8caf-c78fb0669dfc

Closes: #111

### CODE REVIEW

File: src/noshow/api/app_helpers.py [hunk 1]
```diff
@@ -148,6 +154,13 @@ def create_treatment_groups(
         )
     else:
         predictions.loc[:, "treatment_group"] = None
+
+    # Treatment group 2 means excluded from RCT
+    if rct_agendas:
+        predictions.loc[
```
Comment (WoltersEric) [inline src/noshow/api/app_helpers.py:160]: maybe nice to also add this to the unit test as it is quite important that this always works

File: src/noshow/dashboard/connection.py [hunk 2]
```diff
@@ -61,20 +61,22 @@ def get_patient_list(_session: Session, date_input: date) -> List[str]:
     threshold_date = date.today() - timedelta(days=round(MUTE_PERIOD * 30.5))
 
     call_list = _session.execute(
-        select(ApiPrediction.patient_id, func.max(ApiPrediction.prediction))
+        select(
+            ApiPrediction.patient_id,
+            ApiPatient.treatment_group,
```
Comment (WoltersEric) [inline src/noshow/dashboard/connection.py:66]: the query is auto-optimized right? or is this changed order faster?

File: CHANGELOG.md
Comment (WoltersEric) [inline CHANGELOG.md:1]: image
I get this error on the dashboard

**Review by WoltersEric — CHANGES_REQUESTED** [review]
I still get errors on the dashboard

File: src/noshow/dashboard/connection.py (hunk 2 above)
Comment (rubenpeters91) [reply src/noshow/dashboard/connection.py:66]: I think it's auto-optimized, but it makes more sense for me to order them like a query. And I had to add a little bit for the treatment group anyways

File: src/noshow/api/app_helpers.py (hunk 1 above)
Comment (rubenpeters91) [reply src/noshow/api/app_helpers.py:160]: Good point, added two unit tests

**Review by WoltersEric — APPROVED** [review]
Looks good to me!

**Pull request merged**